docs/python3-mac-installation.md

```diff
@@ -0,0 +1,15 @@
+# Latest Python3 installation on macOS
+
+The default Python version (Python 2.x) comes with macOS is not supported. Below are the instructions to install latest Python3 on macOS.  
+
+_Note:_ This guide does not cover `homebrew` based Python3 installation.
+
+1. Navigate to the [official Python download page for macOS](https://www.python.org/downloads/macos/).
+2. Click on the latest stable Python release.
+3. Under the files tab find the list of installers, click and download macOS appropriate installer locally.
+   ![installation_list.png](images/installation_list.png)
```

Comment:
We don't include screenshots unless the text can't make it clear how to do something in the UI. This is because it creates more maintenance (and it tougher if translation is done).

I don't see the reason for including this.